### ISSUE TITLE
warning about potentially uninitialized variable in drfuzz_mutator.c

### ISSUE THREAD

**Issue by maddin200** [DynamoRIO/drmemory#1928]
drfuzz\drfuzz_mutator.c line 228 
map_size can be uninitialized

**Comment by derekbruening** [comment]
No, technically it is never uninitialized (!res will short-circuit first).  But it is best to make it more robust to future changes in the code.  Did some compiler give a warning or something?

**Comment by maddin200** [comment]
just using cppcheck

**Comment by Artem-OSSRevival** [comment]
Hi there, I’m Artem from [OSS Revival](https://github.com/Project-OSS-Revival). We specialize in revitalizing and maintaining open-source projects for long-term stability. In fact, this is not a real problem. As mentioned, the initialization is done above in the code. However, I can suggest adding a standard initialization of the variable to zero. Just for convenience and to improve the quality of the code. If you don't mind, I can do it and add the Pull Request.

**Comment by derekbruening** [comment]
> Hi there, I’m Artem from [OSS Revival](https://github.com/Project-OSS-Revival). We specialize in revitalizing and maintaining open-source projects for long-term stability. In fact, this is not a real problem. As mentioned, the initialization is done above in the code. However, I can suggest adding a standard initialization of the variable to zero. Just for convenience and to improve the quality of the code. If you don't mind, I can do it and add the Pull Request.

Sure, happy to take a Pull Request.

Could you say more about what OSS Revival is able to offer?

**Comment by Artem-OSSRevival** [comment]
> Sure, happy to take a Pull Request.
> 
> Could you say more about what OSS Revival is able to offer?

Sure. We are just starting our work and engaged in the support and development of projects that are valuable for the community and have a significant code, but the enthusiasm of the developers is no longer as great as before :) This is necessary so that the existing code receives updates, improves, and is developed further. There will be more information about us (now the page on the creation/update): https://project-oss-revival.github.io

**Comment by Artem-OSSRevival** [comment]
As for Cppcheck warnings. We did a little research and found that the old version of Cppcheck is required to reproduce these warnings, approximately the Issue creation time (v1.76) and before v2.8. In new Cppcheck versions, this Issue was not reproduced, nevertheless, the Pull Request was created to make the code more robust.

**Comment by derekbruening** [comment]
> > Sure, happy to take a Pull Request.
> > Could you say more about what OSS Revival is able to offer?
> 
> Sure. We are just starting our work and engaged in the support and development of projects that are valuable for the community and have a significant code, but the enthusiasm of the developers is no longer as great as before :) This is necessary so that the existing code receives updates, improves, and is developed further.
> 
> As of today, we're focusing on fixing reported issues in widely used packages across major Unix distributions. This includes triaging, fixing, and testing to keep those packages working. We're also starting to maintain packages in Debian that are no longer maintained in the downstream and plan to expand to upstream maintenance, to make sure those packages are still working in the newer versions. As the team grows, we'll begin working on new features that are in demand. If you have any other issues or needs with other packages, let us know, we'll be happy to help 🙂. https://project-oss-revival.github.io/

Thank you for reaching out.  While there are a number of aspects of Dr. Memory that could use help, probably the most significant issue confronting Dr. Memory is on Windows where the DynamoRIO private loader has been failing on recent Windows versions, leading to Dr. Memory failing to start up.  Does any of your work by any chance include Windows?

**Comment by Artem-OSSRevival** [comment]
> Does any of your work by any chance include Windows?
Currently focused on Linux only :)

**Comment by Artem-OSSRevival** [comment]
> Does any of your work by any chance include Windows?

At the moment focused on Linux. Maybe in the future, when the team grows up...
